README.md

```diff
@@ -301,6 +301,13 @@ login data:
 * User: www-data
 * Password: ilias
 
+You can change the password in the **doil** config file `setup/conf/doil.conf`.
+Before installing **doil** change the default password to your password.
+If doil is already installed you can change the password by `doil mail:change-password <password>`.
+Please ensure you also update the password in your actual installed
+**doil** config `/etc/doil/doil.conf`, this would protect you to set the passwort again after a doil update.
```

Comment:
_"... this would protect you from having to set the password again..."_ sounds a bit better to me

Reply:
Alternatively: _"..., so you don't have to set the password again ..."_

Reply:
Done!